src/ProgressMeter/ProgressMeter.jl

```diff
@@ -111,4 +111,12 @@ function durationstring(nsec)
     hhmmss
 end
 
+# issue #31: isnothing require Julia 1.1
+# copy-over from
+# https://github.com/JuliaLang/julia/blob/0413ef0e4de83b41b637ba02cc63314da45fe56b/base/some.jl
+if !isdefined(Base, :isnothing)
+    isnothing(::Any) = false
+    isnothing(::Nothing) = true
+end
```

Comment:
`isnothing(x)` can also just be replaced with `x === nothing` for julia 1.0.  This is fine though.

Reply:
Yeah, I thought about it. But then I thought that it might be better to have this for the symmetry with `isnan`.